src/spikeinterface/preprocessing/motion.py

```diff
@@ -0,0 +1,350 @@
+import time
+from pathlib import Path
+
+import numpy as np
+import json
+import copy
+
+from spikeinterface.core import get_noise_levels, fix_job_kwargs
+from spikeinterface.core.core_tools import SIJsonEncoder
+
```

Comment:
@cwindolf @yger 
I have started some preset of parameters for the a high level function.
Could you have look and make some feedback ?